services/venice-router/src/main/java/com/linkedin/venice/router/streaming/VeniceChunkedResponse.java

```diff
@@ -246,7 +246,7 @@ public boolean whetherToSkipMessage(Object msg, ChannelPromise promise) {
               "Unexpected response status: " + status + ", and only non 200 status is expected here");
         }
       }
-      throw new VeniceException("Unexpected message type received: " + msg.getClass());
+      throw new VeniceException("Unexpected message type received: " + (msg == null ? "null" : msg.getClass()));
```

Comment:
It used to be possible prior to the fix, and it resulted in a NPE bubbling up from this line, rather than a `VeniceException`.

With the fix, I believe it's not possible for a null to come in, nor for this line to ever be reached at all. So this is just defensive code (as it was always intended to be, AFAICT), but just ensuring that it can't throw a NPE anymore if a future regression were to shove a null `msg` down here.